cmd/influxd/server_integration_test.go

```diff
@@ -433,9 +476,9 @@ func runTestsData(t *testing.T, testName string, nodes Cluster, database, retent
 			expected: `{"results":[{"series":[{"name":"logs","columns":["time","event"]}]}]}`,
 		},
 		{
-			name:     "missing measurement with `GROUP BY *`",
-			query:    `select load from "%DB%"."%RP%".missing group by *`,
-			expected: `{"results":[{"error":"measurement not found: \"%DB%\".\"%RP%\".\"missing\""}]}`,
+			name:          "missing measurement with `GROUP BY *`",
+			query:         `select load from "%DB%"."%RP%".missing group by *`,
+			expectPattern: `.*measurement not found: .*missing.*`,
```

Comment:
I don't follow you. For the query `select load from "%DB%"."%RP%".missing group by *` what would come back?